src/main/resources/application.properties

```
@@ -5,5 +5,11 @@ service.sampletracking.location.user=${SAMPLE_TRACKING_LOCATION_USER:my_email@ex
 service.sampletracking.auth.user=${SAMPLE_TRACKING_AUTH_USER:qbic}
 service.sampletracking.auth.password=${SAMPLE_TRACKING_AUTH_PASSWORD:astrongpassphrase!}
 
+# Properties for the openBIS LIMS application server
+service.openbis.api-server-url=${OPENBIS_SERVER_URL:}
+service.openbis.user=${OPENBIS_USER:}
+service.openbis.password=${OPENBIS_PASSWORD:}
+# openBIS application server timeout in milliseconds
+service.openbis.server-timeout=${OPENBIS_TIMEOUT:10000}
 # File path to the text file that stores the date of the last search for updated samples
-service.last-update.file=${LAST_UPDATE_FILE}
\ No newline at end of file
+service.last-update.file=${LAST_UPDATE_FILE}
```

Why is this in `service`? 
```suggestion
log.file.executions=${LAST_UPDATE_FILE}
```